rocky/tools/templatetags/ooi_extra.py

```diff
@@ -99,3 +100,14 @@ def clearance_level(ooi: OOI) -> ScanLevel:
 @register.filter
 def ooi_type(reference_string: str) -> str:
     return Reference.from_str(reference_string).class_
+
+
+@register.filter
+def get_date(date_str: str) -> datetime:
+    return datetime.fromisoformat(date_str)
```

Comment:
Return type indicates that this is a date time type, not a date.

```suggestion
@register.filter
def get_date(date_str: str) -> date:
    return datetime.fromisoformat(date_str).date
```

Reply:
@madelondohmen is the idea here to return a datetime object? if yes, we should make this function name: get_datetime instead. The function name says to return a date only object as that was the intention or renaming the function to return indeed what it was for .

Reply:
fixed in @707c94c